packages/framework-integration-tests/integration/helper/file-helper.ts

```diff
@@ -38,3 +39,14 @@ export const fileExists = existsSync
 export const dirContents = readdirSync
 
 export const sandboxPathFor = (sandboxName: string): string => sandboxName + '-integration-sandbox' // Add the suffix to make sure this folder is gitignored
+
+export const pidForSandboxPath = (sandboxPath: string): string => path.join(sandboxPath, 'local_provider.pid')
```

Comment:
We are hardcoding `local_provider` name here. Ideally, the name for the file would be passed as a parameter (at least the prefix) so that we can use these functions for other things. Anyway, we will refactor this if needed when the time comes